pages/common/tmux.md

```diff
@@ -19,17 +23,13 @@
 
 `tmux a`
 
-- Attach to a named session:
-
-`tmux a -t {{name}}`
-
```

Comment:
I think this command should keep. `tmux a -t {{name}}` is a quite useful command in daily use.

Reply:
Seconded. I think we really ought to keep the `Start a new named tmux session` one too.

Reply:
Okay, thanks for the feedback. Unfortunately, if I do retain the above two commands, I will not be able to add any more of my examples as this file has already exceeded the limit of 8 examples. Is it okay if I still add my examples as I believe that knowing how to split the pane into two terminals is a vital skill to know.

Reply:
Hrm, I see. That is an awkward one. The thing is that we generally try to keep pages shorter, because they aren't meant to be an _exhaustive_ list of commands etc.

A keyboard shortcut sounds like the kind of thing that would fit well on a cheat sheet.

Reply:
Yes, I understand, so I guess I should just close the pull request?

Reply:
Yes